Data/Sys/Profiles/Wiimote/Wii Remote With MotionPlus Pointing.ini

```diff
@@ -0,0 +1,86 @@
+IMUGyroscope/Dead Zone = 3.0000000000000000
+IMUGyroscope/Pitch Up = `Gyro Pitch Up`
+IMUGyroscope/Pitch Down = `Gyro Pitch Down`
+IMUGyroscope/Roll Left = `Gyro Roll Left`
+IMUGyroscope/Roll Right = `Gyro Roll Right`
+IMUGyroscope/Yaw Left = `Gyro Yaw Left`
+IMUGyroscope/Yaw Right = `Gyro Yaw Right`
+IMUIR/Enabled = True
+IMUIR/Total Yaw = 20.000000000000000
+Extension/Attach MotionPlus = `Attached MotionPlus`
+Extension = `Attached Extension`
+Nunchuk/Buttons/C = `Nunchuk C`
+Nunchuk/Buttons/Z = `Nunchuk Z`
+Nunchuk/Stick/Up = `Nunchuk Y+`
+Nunchuk/Stick/Down = `Nunchuk Y-`
+Nunchuk/Stick/Left = `Nunchuk X-`
+Nunchuk/Stick/Right = `Nunchuk X+`
+Nunchuk/Stick/Calibration = 99.03 96.27 97.29 102.30 108.95 101.86 99.22 100.30 105.52 100.70 99.97 101.77 103.68 99.30 95.07 94.67 98.02 93.32 92.44 95.21 97.31 91.13 88.89 90.15 95.15 92.96 92.37 95.43 100.26 96.04 93.30 94.24
+Nunchuk/Stick/Center = 0.00 0.00
+Nunchuk/Swing/Center = 0.00 0.00
+Nunchuk/Tilt/Center = 0.00 0.00
+Nunchuk/IMUAccelerometer/Up = `Nunchuk Accel Up`
+Nunchuk/IMUAccelerometer/Down = `Nunchuk Accel Down`
+Nunchuk/IMUAccelerometer/Left = `Nunchuk Accel Left`
+Nunchuk/IMUAccelerometer/Right = `Nunchuk Accel Right`
+Nunchuk/IMUAccelerometer/Forward = `Nunchuk Accel Forward`
+Nunchuk/IMUAccelerometer/Backward = `Nunchuk Accel Backward`
+Classic/Buttons/A = `Classic A`
+Classic/Buttons/B = `Classic B`
+Classic/Buttons/X = `Classic X`
+Classic/Buttons/Y = `Classic Y`
+Classic/Buttons/ZL = `Classic ZL`
+Classic/Buttons/ZR = `Classic ZR`
+Classic/Buttons/- = `Classic -`
+Classic/Buttons/+ = `Classic +`
+Classic/Buttons/Home = `Classic HOME`
+Classic/Left Stick/Up = `Classic Left Y+`
+Classic/Left Stick/Down = `Classic Left Y-`
+Classic/Left Stick/Left = `Classic Left X-`
+Classic/Left Stick/Right = `Classic Left X+`
+Classic/Left Stick/Calibration = 93.16 93.84 98.35 102.71 111.99 107.78 107.86 102.30 99.38 100.38 103.49 104.51 103.69 100.24 100.77 97.60 96.10 96.30 97.05 101.76 101.95 102.12 100.96 103.76 111.12 104.34 102.04 103.75 109.80 101.03 97.01 94.36
+Classic/Left Stick/Center = 0.00 0.00
+Classic/Right Stick/Up = `Classic Right Y+`
+Classic/Right Stick/Down = `Classic Right Y-`
+Classic/Right Stick/Left = `Classic Right X-`
+Classic/Right Stick/Right = `Classic Right X+`
+Classic/Right Stick/Calibration = 103.39 100.72 102.00 107.50 115.29 111.04 111.22 104.39 102.04 103.26 103.11 107.06 115.96 106.19 101.54 97.22 96.13 96.82 97.45 102.03 111.57 107.21 107.13 111.34 110.21 111.34 110.13 113.26 110.87 106.51 106.42 110.58
+Classic/Right Stick/Center = 0.00 0.00
+Classic/Triggers/Dead Zone = 5.0000000000000000
+Classic/Triggers/L = `Classic L`
+Classic/Triggers/R = `Classic R`
+Classic/Triggers/L-Analog = `Classic L-Analog`
+Classic/Triggers/R-Analog = `Classic R-Analog`
+Classic/D-Pad/Up = `Classic Up`
+Classic/D-Pad/Down = `Classic Down`
+Classic/D-Pad/Left = `Classic Left`
+Classic/D-Pad/Right = `Classic Right`
+Guitar/Stick/Center = 0.00 0.00
```

Comment:
These "Center" entries could be removed.
There's some above too.

Reply:
Why?

Reply:
Because "0 0" is the default. And the "Guitar" and such are not relevant in this profile. :P
I thought I fixed the default "Center"s being saved but apparently not.